<launch>
    <node name="emlid_nmea" pkg="emlid_reach_tcp" type="emlid_nmea.py" output="screen">
    </node>
</launch>